piyushgarg-dev/piyushgargdev-nextjs
# fix: Second card image in `My Gears` tab and padding between cards and heading

## What does this PR do?

Fixes #1532

## Type of change

Updated the second card's image to match the size of the other cards and added padding between the Gears heading and the cards.

![Piyush Garg - Gears - Brave 6_27_2024 1_55_41 AM](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/156830331/76790001-4a1f-48e5-b185-c3d0e0ae591c)

## How should this be tested?
Navigate to the My Gears tab, where you will notice that the second card's image has been fixed, and padding has been added between the Gears heading and the cards.
